k8s/plugin/README.md

```diff
@@ -113,35 +113,46 @@ Options:
 Pool pool pool-1-kworker1 labelled successfully. Current labels: {"openebs.io/created-by": "operator-diskpool", "zone-us": "east-1"}
```

Comment:
```suggestion
Pool pool pool-1-kworker1 labelled successfully. Current labels: {"zone-us": "east-1"}
```

Reply:
This can not be deleted. Its  can be taken later